src/main/java/com/spotify/reaper/service/RepairRunner.java

```diff
@@ -158,8 +157,11 @@ public void run() {
         case PAUSED:
           context.repairManager.scheduleRetry(this);
           break;
+        case ERROR:
+          resumeAfterError();
```

Comment:
This would behave weirdly. If there's only one thread (3 nodes, RF=3), this should never be reached. ERROR would be set, and run() would never be called again. See my other comment. I don't think we should auto-restart ERROR. Certainly not in the case where ERROR gets set from RepairRunner::repairSegment. Because that one we know is unrecoverable. The catch clause in run() is just for unforeseen errors. They should never happen (they do, but we should figure out why and patch those gaps). So retrying from there... Maybe. But probably rather not.